public/javascripts/SVLabel/css/svl-modal.css

```diff
@@ -87,9 +87,46 @@
     font-weight: normal;
 }
 
+.modal-foreground .col-right {
+    text-align: center;
+}
+
+.modal-foreground .btn {
```

Comment:
Can you change this to blue button? Again, I fear your change will affect other modal windows' appearance.